### PR TITLE
Add dwell events to geofences docs

### PR DESCRIPTION
**Pull request by matthewisabel**
## What?
Add dwell events docs

### CODE REVIEW

File: docs/geofences.mdx [hunk 1]
```diff
@@ -35,6 +35,8 @@ Geofences also provides the following events:
 
 - `user.entered_geofence`
 - `user.exited_geofence`
+- `user.dwelled_in_geofence`
+
```
Comment (jsani-radar) [inline docs/geofences.mdx:39]: nit: extra `\n`

File: docs/geofences.mdx [hunk 2]
```diff
@@ -164,3 +167,13 @@ Turn on _Geofence Stop Detection_ at a project level on the [Settings page](http
 When _Buffer Entries_ is on, Radar will use the accuracy of the location reported to determine overlap with the geofence boundary to generate a geofence entry event. See the [confidence and accuracy](/geofences#confidence-and-accuracy) for more on this detection logic. When off, the location reported will need to fall within the boundaries of the geofence to trigger the entry event.
 
 When _Buffer Exits_ is on, Radar will use the accuracy of the location reported as a buffering distance from the geofence edge before generating a geofence exit event. This buffering minimizes premature exit detection.  When off, the first location reported falling outside the boundaries of a geofence will trigger the exit event.
+
+## Dwell events
+
+<Alert alertType="info">
+  Dwell events are currently in beta
```
Comment (jsani-radar) [inline docs/geofences.mdx:174]: nit: trailing `.`

File: docs/geofences.mdx [hunk 3]
```diff
@@ -164,3 +167,13 @@ Turn on _Geofence Stop Detection_ at a project level on the [Settings page](http
...
+</Alert>
+
+Dwell events are triggered when a geofence is occupied for more time than a given threshold (in minutes). When `user.dwelled_in_geofence` is enabled for a project, dwell time thresholds can be set on individual geofences via import (by adding a `dwellThreshold` column) or globally across all geofences by contacting your account representative. Dwell events will trigger on the first track call that occurs after the dwell threshold is exceeded. As a result, increasing tracking frequency when inside of a geofence will decrease the time gap from when the threshold is first exceed and the event is fired.
+
+Turn on `user.dwelled_in_geofence` events at the project level on the [Settings page](https://radar.com/dashboard/settings), under _Geofences_.
```
Comment (jsani-radar) [inline docs/geofences.mdx:179]: this href can be anchored to the geofences form

File: docs/geofences.mdx [hunk 4]
```diff
@@ -157,10 +159,20 @@ You may decide to ignore some events based on confidence levels.
...
 
 When _Buffer Exits_ is on, Radar will use the accuracy of the location reported as a buffering distance from the geofence edge before generating a geofence exit event. This buffering minimizes premature exit detection.  When off, the first location reported falling outside the boundaries of a geofence will trigger the exit event.
+
+## Dwell events
+
+<Alert alertType="info">
+  Dwell events are currently in beta.
+</Alert>
+
+Dwell events are triggered when a geofence is occupied for more time than a given threshold (in minutes). When `user.dwelled_in_geofence` is enabled for a project, dwell time thresholds can be set on individual geofences via import (by adding a `dwellThreshold` column) or globally across all geofences by contacting your account representative. Dwell events will trigger on the first track call that occurs after the dwell threshold is exceeded. As a result, increasing tracking frequency when inside of a geofence will decrease the time gap from when the threshold is first exceed and the event is fired.
```
Comment (bschoeneweis) [inline docs/geofences.mdx:176]: nit: can add API here as well

`dwell time thresholds can be set on individual geofences via import (by adding a dwellThreshold column), the API, or globally across all geofences by contacting your account representative.`

**Review by bschoeneweis — APPROVED** [review]
Looks good 👍.  I'll give the green flag, but still waiting for @samfader's feedback

**Comment by samfader** [comment]
@matthewisabel I could see this potentially being misleading:

> Dwell events are triggered when a geofence is occupied for more time than a given threshold (in minutes)

This is...sort of true, right? It has to be occupied but also the user needs to either open the app/interact with it (foreground) or have background tracking enabled for it to be 100% true. Curious what you think...

**Comment by matthewisabel** [comment]
@samfader I clarify how it works in that paragraph two sentences later, it's just a lot of concepts to work into an opening sentence without confusing the reader (feature must be enabled, threshold must be set in minutes, subsequent track call must occur). If you think there's a better wording lmk, trying to explain it as concisely as possible just a challenge.

File: docs/geofences.mdx (hunk 4 above)
Comment (matthewisabel) [reply docs/geofences.mdx]: Swapped, thanks. You can also do this comment as a `suggestion` so I can just one click add in the future: https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review.

Comment (bschoeneweis) [reply docs/geofences.mdx]: Oh nice, I did not know that.  Solid tip

**Pull request merged**